server/command.go

```diff
@@ -10,21 +10,21 @@ import (
 	"github.com/mattermost/mattermost-server/plugin"
 )
 
-const helpText = "###### Mattermost JIRA Plugin - Slash Command Help\n" +
-	"* `/jira connect` - Connect your Mattermost account to your JIRA account\n" +
-	"* `/jira disconnect` - Disonnect your Mattermost account to your JIRA account\n" +
-	"* `/jira instance` - Manage JIRA instances connected to Mattermost\n" +
-	"  * `list` - List known JIRA instances\n" +
-	"  * `select <key or number>` - Select a known instance as current\n" +
-	"  * `add server <URL>` - Add a JIRA Server instance\n" +
-	"  * `add cloud` - Add a JIRA Cloud instance\n" +
+const helpText = "###### Mattermost Jira Plugin - Slash Command Help\n" +
+	"* `/jira connect` - Connect your Mattermost account to your Jira account and subscribe to events\n" +
+	"* `/jira disconnect` - Disonnect your Mattermost account from your Jira account\n" +
+	"* `/jira instance [add/list/select]` - Manage connected Jira instances. Must have System Admin role in Mattermost\n" +
+	"  * `/jira instance add server <URL>` - Connect a Jira Server instance to Mattermost\n" +
+	"  * `/jira instance add cloud` - Connect a Jira Cloud instance to Mattermost\n" +
+	"  * `/jira instance list` - List connected Jira instances\n" +
+	"  * `/jira instance select <key or number>` - Select the active Jira instance. At most one active Jira instance is currently supported\n" +
```

Comment:
s/key/URL/

Reply:
I would change to `<number or URL>` to match the order in the table.

Also, someone should probably change the actual table column headings to be correct (I'll do that in my PR). :)